### PR TITLE
Updating docs for 1.1.1 patch release notes + versions

### PR DESCRIPTION
**Pull request by stevshap**
Signed-off-by: stshapir <steven.shapiro@oracle.com>

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ weight: 13
 draft: false
 ---
 
+### v1.1.1
+Fixes:
+- Elasticsearch and Keycloak images were updated to address vulnerabilities.
```
Comment (mgianatagh) [inline content/en/docs/releasenotes/_index.md:10]: @markxnelson is it ok to say this?

Comment (stevshap) [reply content/en/docs/releasenotes/_index.md:10]: We said it in our 1.0.4 release

Comment (wmhopkins) [reply content/en/docs/releasenotes/_index.md:10]: Yes, we're not revealing any specifics about the vulnerabilities.

Probably better to call them CVEs, though.

Comment (rosemarymarano) [reply content/en/docs/releasenotes/_index.md:10]: Why is it better to call them CVEs? Is that a more accurate description of what was addressed in these updates?

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -5,6 +5,14 @@ weight: 13
...
+- Updated WebLogic Kubernetes Operator image to 3.3.7.
+- Minor bug fixes including updated Elasticsearch logging to avoid type collisions.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:12]: including updated Elasticsearch -> including updating Elasticsearch

**Review by rosemarymarano — COMMENTED** [review]
Minor edit, otherwise ok.

**Pull request merged**